h/paginator.py

```diff
@@ -92,3 +92,7 @@ def wrapper(context, request):
             'page': page,
         }
     return wrapper
+
+
+def paginate_es(request, result, page_size=PAGE_SIZE):
```

Comment:
I don't really see what this is adding.

I'd suggest adding the `PAGE_SIZE` default to the `paginate` function and then just using that from the view function.